docs/CHANGELOG.md

```diff
@@ -0,0 +1,147 @@
+- Remove `getCellActions` prop. Check [here](https://github.com/adazzle/react-data-grid/pull/1845) on how to migrate
+- Remove `react-data-grid-addons` package.
+- Remove subrow props (`getSubRowDetails`, `onCellExpand`, `onDeleteSubRow`, and `onAddSubRow`). Check [here](https://github.com/adazzle/react-data-grid/pull/1853) on how to migrate
+- Remove `cellContentRenderer`
+- `column.editor`, `rowRenderer`, and `headerRenderer` no longer support `ReactElement`
+- `column.formatter` no longer receives a `value` prop.
+- Remove `column.events` prop
+- Replace `rowGetter` and `rowsCount` props with a single `rows` prop
+- Rename `minHeight` to `height`
+- Rename `minWidth` to `width`
+- Rename `RowsContainer` to `rowsContainer`
+- Rename `onGridSort` to `onSort`
+- Rename `onGridRowsUpdated` to `onRowsUpdate`
+
+## next -> canary
+- Column resize now resizes all the cells which means re-rendering the whole grid when resizing columns.
+- Add `column.cellContentRenderer`. This is an opt-in feature, so it should not break anything. More info in [here](https://gist.github.com/nstepien/090298c3c2d94324cb332c33d82fdcfb)
+- `ReactDataGrid` component is renamed to `DataGrid`
+- Remove `setScrollLeft` from Row and Cell renderers. `setScrollLeft` instance method was previously required on custom Cell and Row renderers and it can be safely removed now. More info in [here](https://github.com/adazzle/react-data-grid/pull/1793)
+- Drop ImmutableJS support
+- Drop dynamic height row support. This was not an official feature but it was still possible to implement dynamic rows via custom Row renderer. This was a buggy feature so it has been removed.
+- Remove `formatter.isScrolling` prop. `isScrolling` prop was introduced to improve performance of expensive formatters. This is no longer required as the internals of the grid were rewritten to improve performance
+- Add summary rows [1773](https://github.com/adazzle/react-data-grid/pull/1773)
+- Remove row reordering support. This was a buggy feature so it has been removed. 
+- Remove toolbar prop. Check [here](https://github.com/adazzle/react-data-grid/pull/1769) on how to migrate.
+- Remove internal `sortColumn` and `sortDirection` states. Sorting is now only controlled by `sortColumn` and `sortDirection` props. Check [here](https://github.com/adazzle/react-data-grid/pull/1768) on how to migrate.
+- Improve scrolling performance
+- Improve frozen columns performance
+- Re-implement row selection. Previous row selection API has been replaced by the new `selectedRows` and `onSelectedRowsChange` props. A new `SelectColumn` component is now needed to add checkbox column. Check [here](https://github.com/adazzle/react-data-grid/pull/1762) on how to migrate.
+- Add `renderBatchSize` prop [1779](https://github.com/adazzle/react-data-grid/pull/1779)
+- Remove bootstrap dependency and re-implement grid css [1780](https://github.com/adazzle/react-data-grid/pull/1780)
+
+## master -> next
```

Comment:
```suggestion
## master -> alpha
```
`next` is the branch name, while `alpha` is the npm tag, should probably use it instead. And rename the next branch maybe.